### PR TITLE
Feature/create architecture

### PR DESCRIPTION
**Pull request by alia-peterson**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
- Added folder/file architecture
- Created Header, Form, Container, and Card components 
- Added basic header elements and created form dropdowns with cities and states populated from data file

## *What are the relevant tickets?*
none

## *Bugs:*
No known bugs

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Comment by alia-peterson** [comment]
[As a user, when I visit the website, I will see a form where I can select a state and city that I want information about.](https://trello.com/c/jcLXbLY6/15-as-a-user-when-i-visit-the-website-i-will-see-a-form-where-i-can-select-a-state-and-city-that-i-want-information-about)

**Comment by alia-peterson** [comment]
[As a user, when I visit the website, I will see a form where I can select a state and city that I want information about.](https://trello.com/c/jcLXbLY6/15-as-a-user-when-i-visit-the-website-i-will-see-a-form-where-i-can-select-a-state-and-city-that-i-want-information-about)

File: src/components/App/App.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import data from '../../data/location-data'
+
+export default {
+  name: 'App',
+  components: {
+    Header,
+    Form,
+    Container
+  },
+  data: () => ({
```
Comment (tashiad) [inline src/components/App/App.vue:23]: Glad we figured this out in the end! Looking forward to moving on to bringing in the API data instead.

File: src/components/App/App.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
```
Comment (tashiad) [inline src/components/App/App.vue:2]: I think we may not need this div?

File: src/components/Form/Form.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      >{{city.city}}</option>
+    </select>
+    <input type="submit" name="submit" value="Submit">
+  </form>
+</template>
+
+<script>
+export default {
+  name: 'Form',
+  props: {
```
Comment (tashiad) [inline src/components/Form/Form.vue:26]: I'm still a little unclear on why it's necessary to declare the data types in props (and why sometimes it works without). Let's look into this deeper later!

File: src/components/App/App.vue [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  data: () => ({
+    cities: data.cityData.data,
+    states: data.stateData.data
+  }),
+  methods: {
+    debug (input) {
+      console.log(input)
+    }
+  }
```
Comment (dietza) [inline src/components/App/App.vue:31]: Glad that we figure out how to use this approach to print to the console in Vue, but expect we'll want to pull this out sooner than we forget

**Review by tashiad — APPROVED** [review]
Great start! 

Let's remember to check on whether the icons should live in the existing assets folder or if it would make more sense for them to stay in public.

File: src/components/Form/Form.vue [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <form class="">
+    <select class="" name="">
+      <option>Choose a State</option>
+      <option
+        v-for="(state, index) in states"
+        :key="index"
+        :value="state.state"
```
Comment (dietza) [inline src/components/Form/Form.vue:8]: This seems like a great solution at this point - just wondering if there is something more specific we should look to later as an id/key here?

File: src/data/location-data.js [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+        city: 'Thornton',
+        state: 'Colorado',
+        country: 'USA',
+        location: {
+          type: 'Point',
+          coordinates: [
+            -104.97192,
+            39.86804
+          ]
+        },
```
Comment (dietza) [inline src/data/location-data.js:15]: so glad you sorted out the weird error the linter was giving us for this data file, but still curious if we'll need to readdress the formatting when we parse data from the api

**Review by dietza — APPROVED** [review]
- [x] Changes generate no new warnings or errors
- [x] Branch was tested locally without failures
- [n/a] All tests are passing as expected

Thank you for steering this setup, Alia! This was helpful for me to see, and I really appreciate this approach we've landed on for the structure of our single file components.

**Pull request merged**